src/transforms/transforms.ts

```diff
@@ -56,10 +56,18 @@ export function transformChoice(
   const metadata = getLanguageMetadata(choice.language);
   let name = normalizeName(metadata.name, NameType.Interface);
   let schemaType = choice.type;
+  let itemType =
+    //@ts-ignore
```

Comment:
What is this @ts-ignore for?

Reply:
By definition `choice.choiceType.type` is defined as StringSchema. So, TS will complain that the condition will never be true. So, added this line to override that